### PR TITLE
[TASK] update version in ext_emconf.php

### PR DESCRIPTION
**Pull request by ihorko-git**

### CODE REVIEW

File: Configuration/Yaml/PxaFormEditor.yaml [hunk 1]
```diff
@@ -6,16 +6,14 @@ TYPO3:
 
           formEditor:
             translationFile:
-              0:  'EXT:form/Resources/Private/Language/Database.xlf'
-              1:  'EXT:pxa_form_enhancement/Resources/Private/Language/locallang_db.xlf'
+              0: EXT:form/Resources/Private/Language/Database.xlf
+              1: EXT:pxa_form_enhancement/Resources/Private/Language/locallang_db.xlf
             translationFiles:
```
Comment (ihorko-git) [inline Configuration/Yaml/PxaFormEditor.yaml:12]: https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Breaking-87009-UseMultipleTranslationFilesByDefaultInEXTform.html

File: Configuration/Yaml/PxaFormEditor.yaml [hunk 2]
```diff
@@ -24,45 +22,44 @@ TYPO3:
...
           finishersDefinition:
             SaveForm:
               formEditor:
-                iconIdentifier: 'ext-pxaformenhancement-finisher-icon'
-                label: 'formEditor.elements.Form.finisher.SaveForm.editor.header.label'
+                iconIdentifier: ext-pxaformenhancement-finisher-icon
+                label: formEditor.elements.Form.finisher.SaveForm.editor.header.label
                 predefinedDefaults:
                   options:
                     name: ''
```
Comment (ihorko-git) [inline Configuration/Yaml/PxaFormEditor.yaml:65]: if null (empty value without quotes) validators do not work in backend.

File: composer.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     }
   },
   "replace": {
-    "typo3-ter/pxa_form_enhancement": "self.version"
```
Comment (ihorko-git) [inline composer.json:12]: Looks like this package does not exist.

File: ext_emconf.php [hunk 4]
```diff
@@ -21,7 +21,7 @@
     'uploadfolder' => '0',
     'createDirs' => '',
     'clearCacheOnLoad' => 1,
-    'version' => '3.2.7',
+    'version' => '3.3.0',
```
Comment (ihorko-git) [inline ext_emconf.php:24]: Fixed version. Probably tag v3.3 should be deleted.

**Review by mabolek — APPROVED** [review]

**Pull request merged**